packages/serverless-contracts/src/contracts/apiGateway/features/lambdaHandler.ts

```diff
@@ -49,6 +49,7 @@ export const getApiGatewayHandler =
     } = OutputType<Contract>,
   >(
     contract: Contract,
+    ajv: Ajv,
```

Comment:
don't you want to make it optional with a default value `new Ajv({ keywords: ['faker'] });`?

1. to minimize the breaking change
2. to simplify the usage if the default ajv configuration is suficient

Reply:
the tradeoff of this is that  Ajv would no longer be an optional peerDeps as planned but is still a dependency.

Reply:
To align with other features (getAxiosRequest, getFetchRequest), I would prefer to introduce a breaking change to align them on the same basis.
The breaking change documentation is quite straightforward, and I really think that it's a good practice for user to have the control on their ajv instance.

What do you think ?